.travis.yml

```diff
@@ -118,6 +118,8 @@ services:
 - postgresql
 - redis-server
 before_install:
+# Add IPv6 interfaces: https://github.com/travis-ci/travis-ci/issues/8361
+- if [ "${TRAVIS_OS_NAME}" == "linux" ]; then sudo sh -c 'echo 0 > /proc/sys/net/ipv6/conf/all/disable_ipv6'; fi
```

Comment:
We only run on Linux, so you can drop the OS check, but otherwise seems fine

Reply:
done